lib/ddr/models/has_admin_metadata.rb

```diff
@@ -61,7 +61,7 @@ module HasAdminMetadata
                predicate: Ddr::Vocab::Asset.workflowState,
                multiple: false
 
-      delegate :publish, :publish!, :unpublish, :unpublish!, :published?, to: :workflow
+      delegate :publish!, :unpublish!, :published?, :publishable?, to: :workflow
```

Comment:
OK, I guess since this is for 3.x beta we don't need to deprecate. Do please add it to release/upgrade notes.